packages/@ourworldindata/grapher/src/controls/ContentSwitchers.scss

```diff
@@ -35,3 +68,30 @@
         }
     }
 }
+
+.Tabs.ContentSwitchers {
+    .Tabs__Tab.ContentSwitchers__OverflowMenuButton {
+        // Make the +X tab a bit narrower
+        padding: 0 8px;
+
+        // Don't show as 'active' when clicked
+        &[data-selected] {
+            background: #fff;
+            color: $light-text;
+
+            // Don't hide the separator line
```

Comment:
This (?) looks a bit odd, with the tiny separator line being in place:
![CleanShot 2025-07-02 at 13 24 23](https://github.com/user-attachments/assets/99a2d886-534d-4aa7-8937-2a7a8ce60d88)

Reply:
yeah, you're right! not sure why I did that. probably a left-over change from a previous version..